sdk/resourcemanager/azure-resourcemanager-containerregistry/src/test/java/com/azure/resourcemanager/containerregistry/RegistryTest.java

```diff
@@ -27,8 +27,8 @@ public abstract class RegistryTest extends ResourceManagerTestProxyTestBase {
 
     public RegistryTest() {
         addSanitizers(
-            new TestProxySanitizer(String.format("$..%s", "uploadUrl"), null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY),
-            new TestProxySanitizer(String.format("$..%s", "logLink"), null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY)
+            new TestProxySanitizer("$..uploadUrl", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY),
+            new TestProxySanitizer("$..logLink", null, REDACTED_VALUE, TestProxySanitizerType.BODY_KEY)
         );
     }
```

Comment:
Perhaps we can put them all under `ResourceManagerTestProxyTestBase`?

Reply:
This 2 is not unambiguously key for secret. They could accidently match to other valid property in JSON. Hence I still put it in ACR instead of Base.

If we see more of them in other module, maybe we will, but with smaller scope e.g. `$.logLink` if it is directly under root of response JSON.